content/zh-cn/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -124,17 +154,176 @@ <h3>Service 和 Label</h3>
+				要创建一个新的 Service 然后暴露给外部流量，我们将使用 <code>expose</code> 命令，并将 NodePort 作为参数。</p>
+				<p><code><b>kubectl expose deployment/kubernetes-bootcamp --type="NodePort" --port 8080</b></code></p>
+				<!--
+				<p>Let's run again the <code>get services</code> subcommand:</p>
+				-->
+				<p>让我们再次运行 <code>get services</code> 子命令：</p>
+				<p><code><b>kubectl get services</b></code></p>
+				<!--
+				<p>We have now a running Service called kubernetes-bootcamp. Here we see that the Service received a unique cluster-IP, an internal port and an external-IP (the IP of the Node).</p>
+				<p>To find out what port was opened externally (for the <tt>type: NodePort</tt> Service) we’ll run the <code>describe service</code> subcommand:</p>
+				-->
+				<p>我们现在有一个运行中的 Service 名为 kubernetes-bootcamp 。这里我们看到 Service 收到了一个唯一的 cluster-IP ，一个内部端口和一个 external-IP (Node 的 IP)。</p>
+				<p>要了解外部打开的端口（对于 <tt>type: NodePort</tt> 的服务），我们需要运行 <code>describe service</code> 子命令：</p>
+				<p><code><b>kubectl describe services/kubernetes-bootcamp</b></code></p>
+				<!--
+				<p>Create an environment variable called <tt>NODE_PORT</tt> that has the value of the Node port assigned:</p>
+				-->
+				<p>创建一个名为 <tt>NODE_PORT</tt> 的环境变量，它的值为分配的 Node 端口：</p>
+				<p><code><b>export NODE_PORT="$(kubectl get services/kubernetes-bootcamp -o go-template='{{(index .spec.ports 0).nodePort}}')"</b></code><br />
+				<code><b>echo "NODE_PORT=$NODE_PORT"</b></code></p>
+				<!--
+				<p>Now we can test that the app is exposed outside of the cluster using <code>curl</code>, the IP address of the Node and the externally exposed port:</p>
+				-->
+				<p>现在我们可以使用 <code>curl</code> 、Node 的 IP 地址和对外暴露的端口，来测试应用是否已经暴露到了集群外部：</p>
+				<p><code><b>curl http://"$(minikube ip):$NODE_PORT"</b></code></p>
+				<!--
+        {{< note >}}<p>If you're running minikube with Docker Desktop as the container driver, a minikube tunnel is needed. This is because containers inside Docker Desktop are isolated from your host computer.<br>
+				<p>In a separate terminal window, execute:<br>
+				<code><b>minikube service kubernetes-bootcamp --url</b></code></p>
+				<p>The output looks like this:
+				<pre><b>http://127.0.0.1:51082<br>!  Because you are using a Docker driver on darwin, the terminal needs to be open to run it.</b></pre></p>
+				<p>Then use the given URL to access the app:<br>
+				<code><b>curl 127.0.0.1:51082</b></code></p>
+				{{< /note >}}
+				<p>And we get a response from the server. The Service is exposed.</p>
+				-->
+				{{< note >}}<p>如果你正在使用 Docker Desktop 作为容器驱动来运行 minikube, 需要 minikube tunnel。这是因为Docker Desktop 内部的容器和宿主机是隔离的。<br>
+				<p>在另一个终端窗口中，执行:<br>
+				<code><b>minikube service kubernetes-bootcamp --url</b></code></p>
+				<p>输入结果如下:
+				<pre><b>http://127.0.0.1:51082<br>!  Because you are using a Docker driver on darwin, the terminal needs to be open to run it.</b></pre></p>
+				<p>然后使用提供的 URL 访问应用：<br>
+				<code><b>curl 127.0.0.1:51082</b></code></p>
+				{{< /note >}}
```

Comment:
```suggestion
				{{< note >}}
				<!--
				<p>If you're running minikube with Docker Desktop as the container driver, a minikube tunnel is needed. This is because containers inside Docker Desktop are isolated from your host computer.<br>
				<p>In a separate terminal window, execute:<br>
				<code><b>minikube service kubernetes-bootcamp --url</b></code></p>
				<p>The output looks like this:
				<pre><b>http://127.0.0.1:51082<br>!  Because you are using a Docker driver on darwin, the terminal needs to be open to run it.</b></pre></p>
				<p>Then use the given URL to access the app:<br>
				<code><b>curl 127.0.0.1:51082</b></code></p>
				{{< /note >}}
				<p>And we get a response from the server. The Service is exposed.</p>
				-->
				<p>如果你正在使用 Docker Desktop 作为容器驱动来运行 minikube，需要 minikube tunnel。这是因为 Docker Desktop 内部的容器和宿主机是隔离的。<br>
				<p>在另一个终端窗口中，执行：<br>
				<code><b>minikube service kubernetes-bootcamp --url</b></code></p>
				<p>输出结果如下：
				<pre><b>http://127.0.0.1:51082<br>!  Because you are using a Docker driver on darwin, the terminal needs to be open to run it.</b></pre></p>
				<p>然后使用提供的 URL 访问应用：<br>
				<code><b>curl 127.0.0.1:51082</b></code></p>
				{{< /note >}}
```

Reply:
Looks like the `{{< note>}}` tag cannot before the en source comment, or else it will rendered on the page.

Reply:
Read zh-cn localization style guide: https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#shortcode

Reply:
🥲Looks like it can't rendered right: https://deploy-preview-43850--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tutorials/kubernetes-basics/expose/expose-intro/

This seems caused by indentations of EN comment, if I remove indentations in `shortcode` block, it will be ok.

Reply:
Yes, you are correct here.